support-frontend/assets/helpers/campaigns.jsx

```diff
@@ -19,17 +19,18 @@ export type CampaignSettings = {
 };
 
 export type Campaigns = {
-  [string]: CampaignSettings,
+  [string]: (goalReached: boolean) => CampaignSettings,
 };
 
 const currentCampaignName = ausMomentEnabled('AU') ? 'au/contribute' : null;
 
 export const campaigns: Campaigns = currentCampaignName ? {
-  // TODO - the rest of the campaign settings
-  [currentCampaignName]: {
+  [currentCampaignName]: (goalReached: boolean) => ({
```

Comment:
The type of this value not the same between the two values of `currentCampaignName`. If it's true, the return is a function, as in the definition, if false, it's an (empty) object like it used to be.

Reply:
good spot! You are clearly a better type checker than flow is

Reply:
ah wait, it is correct actually.
If `currentCampaignName` is `null` then we set `campaigns` to be an empty object.

But - this is all a bit needlessly complicated, I will simplify it

Reply:
I've just pushed a refactor of the campaigns logic, hopefully it's easier to reason about now!